yb_migrate/src/migration/postgres.go

```diff
@@ -375,7 +385,7 @@ func createTableListPatterns(tableList []string) string {
 	var tableListPattern string
 
 	for _, table := range tableList {
-		tableListPattern += fmt.Sprintf("-t %s ", table)
+		tableListPattern += fmt.Sprintf("-t '%s' ", table)
```

Comment:
same question as above

Reply:
Take a look at the generated pg_dump command in case of both case sensitive and non-sensitive cases:
```
pg_dump "postgresql://postgres:password@localhost:5432/dvdrental?sslmode=prefer" --no-blobs --data-only --compress=0 -t 'public.actor' -t 'public.address' -t 'public.category' -t 'public.city' -t 'public.country' -t 'public.customer' -t 'public.film' -t 'public.film_actor' -t 'public.film_category' -t 'public.inventory' -t 'public.language' -t 'public.payment' -t 'public.rental' -t 'public.staff' -t 'public.store' -t 'public.actor_actor_id_seq' -t 'public.address_address_id_seq' -t 'public.category_category_id_seq' -t 'public.city_city_id_seq' -t 'public.country_country_id_seq' -t 'public.customer_customer_id_seq' -t 'public.film_film_id_seq' -t 'public.inventory_inventory_id_seq' -t 'public.language_language_id_seq' -t 'public.payment_payment_id_seq' -t 'public.rental_rental_id_seq' -t 'public.staff_staff_id_seq' -t 'public.store_store_id_seq'  -Fd --file /home/amit.jambure/export-dirs/dvdrental/data --jobs 1
```
And
```
pg_dump "postgresql://postgres:password@localhost:5432/chinook?sslmode=prefer" --no-blobs --data-only --compress=0 -t 'public."Album"' -t 'public."Artist"' -t 'public."Customer"' -t 'public."Employee"' -t 'public."Genre"' -t 'public."Invoice"' -t 'public."InvoiceLine"' -t 'public."MediaType"' -t 'public."Playlist"' -t 'public."PlaylistTrack"' -t 'public."Track"'  -Fd --file /home/amit.jambure/export-dirs/chinook/data --jobs 1
```
Without the enclosing single-quotes, pg_dump doesn't recognise the case-sensitive names.
```
[amit.jambure@dev-server-ajambure ~]$ pg_dump "postgresql://postgres:password@localhost:5432/chinook?sslmode=prefer" --no-blobs --data-only --compress=0  -Fd --file /home/amit.jambure/export-dirs/chinook/data --jobs 1 -t public.'Album'
pg_dump: error: no matching tables were found
[amit.jambure@dev-server-ajambure ~]$ pg_dump "postgresql://postgres:password@localhost:5432/chinook?sslmode=prefer" --no-blobs --data-only --compress=0  -Fd --file /home/amit.jambure/export-dirs/chinook/data --jobs 1 -t "public.'Album'"
pg_dump: error: no matching tables were found
[amit.jambure@dev-server-ajambure ~]$ pg_dump "postgresql://postgres:password@localhost:5432/chinook?sslmode=prefer" --no-blobs --data-only --compress=0  -Fd --file /home/amit.jambure/export-dirs/chinook/data --jobs 1 -t public.Album
pg_dump: error: no matching tables were found
[amit.jambure@dev-server-ajambure ~]$ pg_dump "postgresql://postgres:password@localhost:5432/chinook?sslmode=prefer" --no-blobs --data-only --compress=0  -Fd --file /home/amit.jambure/export-dirs/chinook/data --jobs 1 -t 'public."Album"'
[amit.jambure@dev-server-ajambure ~]$
```

Reply:
ok, looks good